File: Modulo_II/funcionario/JAVA/Funcionario.java
class Funcionario {

    private String nome;
    private String email;

    public Funcionario() {}

    public Funcionario(String nome, String email){

        this.nome = nome;
        this.email = email;
    }

    public void setNome(String nome) { this.nome = nome; }
    public void setEmail(String email) { this.email = email; }

    public String getNome() { return nome; }
    public String getEmail() { return email; }

    public String exibeDados(){

        return "Nome: " + this.nome + " | Email: " + this.email;
    }

    
}

File: Modulo_II/funcionario/PY/Funcionario.py
class Funcionario:

    def __init__(self):
        
        nome = ''
        email = ''

    def __init__(self, nome, email):
        
        self.nome = nome
        self.email = email


    def __str__(self):
        
        return f'Nome: {self.nome} | Email: {self.email}'
